src/MyHealth/SymptomLogContext.spec.tsx

```diff
@@ -132,7 +132,7 @@ describe("SymptomLogProvider", () => {
 })
 
 const CurrentCheckInStatus = () => {
```

Comment:
Maybe not the best name for this dummy component since no check-in logic is present anymore.